requirements/docs.txt

```diff
@@ -0,0 +1,3 @@
+rstcheck
```

Comment:
You forgot to also update `requirements-dev.txt`

Reply:
done